src/vnsw/agent/port_ipc/port_ipc_handler.cc

```diff
@@ -499,7 +499,7 @@ bool PortIpcHandler::GetPortInfo(const string &uuid_str, string &info) const {
 
 bool PortIpcHandler::InterfaceExists(const std::string &name) const {
 #if 0 //WINDOWS-TEMP
-    int indx  = if_nametoindex(name.c_str());
+    int indx  = osspecific_if_nametoindex(name.c_str());
```

Comment:
Yes, the osspecific functions call that msdn function.  These functions were intentionally separated out to prevent any os-specific returns (different structures etc.)